.dockerfiles/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+wait_for() {
+	host="${1}"
+	port="${2}"
+	name="${3}"
+	timeout="${4:-30}"
+
+	printf "Waiting for %s on %s:%s ... " "${name}" "${host}" "${port}"
+
+	i=0
+	while ! nc -z "${host}" "${port}"; do
+		sleep 1
+		if [ "$((++i))" -gt "$timeout" ]; then
```

Comment:
I thought that `$((++i))` was not working with `/bin/sh`. Maybe switch to bash?

Reply:
Also, maybe we could use something like https://github.com/vishnubob/wait-for-it (it has a Debian/Ubuntu package) or https://hub.docker.com/r/waisbrot/wait (It's a docker image) to avoid re-inventing the wheel? Just a suggestion.

Reply:
> I thought that $((++i)) was not working with /bin/sh. Maybe switch to bash?

Yeah it's a Bash thing :)

Reply:
I want to use `/bin/sh` mostly in our CI/CD because I don't really care for pulling in Bash and all its dependencies :) Some ~13MB on Alpine for example

Reply:
> Also, maybe we could use something like https://github.com/vishnubob/wait-for-it (it has a Debian/Ubuntu package) or https://hub.docker.com/r/waisbrot/wait (It's a docker image) to avoid re-inventing the wheel? Just a suggestion.

I _actually_ completely agree; I borrowed this code from elsewhere anyway. But for now this will work. It might get iterated out for sure :)